app/routes.php

```diff
@@ -45,7 +45,7 @@
     }
 
     return $next($request, $response);
-});
+})->add(new Pavlakis\Middleware\Csp\CspMiddleware($container->get('csp.config'), true));
```

Comment:
Maybe add a comment here to show this group has the middleware applied but it is in Report-only (default) state